README.md

```diff
@@ -149,6 +155,45 @@ if err != nil {
 zlintResultSet := zlint.LintCertificateEx(parsed, registry)
 ```
 
+To lint a certificate in the presence of a particular configuration file, you must first construct the configuration and then make a call to `SetConfiguration` in the `Registry` interface.
+
+A `Configuration` may be constructed using any of the following functions:
+
+* `lint.NewConfig(r io.Reader) (Configuration, error)`
+* `lint.NewConfigFromFile(config string) (Configuration, error)`
```

Comment:
nit: The argument to this function is called `path`, not `config`, in the actual function definition.